### PR TITLE
docs(rxTags): FRMW-294 Migrate rxTags

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-294

- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

**Comment by ssalinas24** [comment]
@CITguy DO NOT REVIEW YET! I got pulled away during this PR, and forgot to make a change before my commit. I will make the corrections.

File: src/rxTags/rxTags.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+ * # rxTags component
+ *
+ * A component used to apply predetermined descriptions to an entity.
+ *
+ * ## Directives
+ * * {@link encore.ui.rxTags:rxTags rxTags}
+ *
+ * ## Filters
+ * * {@link encore.ui.rxTags:xor xor}
+ *
```
Comment (ssalinas24) [inline src/rxTags/rxTags.js]: @CITguy I am not sure if these links on line: 10 and line: 13 are correct, and I may need your assistance if they are not. Please let me know.

Comment (CITguy) [inline src/rxTags/rxTags.js]: should be `{@link rxTags.directive:rxTags rxTags}` and `{@link rxTags.filter:xor xor}`

Comment (ssalinas24) [inline src/rxTags/rxTags.js]: I tried that previously and get the following errors:

```
WARNING: api/rxTags (defined in src/rxTags/rxTags.js) points to a non existing page "api/rxTags.directive:rxTags"!
WARNING: api/rxTags (defined in src/rxTags/rxTags.js) points to a non existing page "api/rxTags.filter:xor"!
```

Comment (ssalinas24) [inline src/rxTags/rxTags.js]: Ok I got it. I update the @name items as well.

**Comment by ssalinas24** [comment]
@CITguy I fixed the links squashed and submitted. Now this should be ready for review. Thank you for the assistance. I think my previous attempts had left out the .directive and .filter in the ```@name``` parameters.

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**